docs/rebuild_admin.rst

```diff
@@ -0,0 +1,407 @@
+three related files from the *Application Server* to the *Admin Workstation*.
+
+To retrieve these files, use the commands:
+
+.. code:: sh
+
+   cd ~/Persistent/securedrop/install_files/ansible-base
+   ssh app sudo tar -c -C /var/lib ssl/  | tar xvf - 
+
+These commands will create a directory named 
+``~/Persistent/securedrop/install_files/ansible-base/ssl``
+on the *Admin Workstation*, containing your instance's SSL certificate, 
+certificate key, and chain file. When prompted for the names of these files
+during the next step, you should specify them relative to the 
+``install_files/ansible-base`` directory, i.e. as ``ssl/mydomain.crt``.
+
+Step 5: Configure and back up the application
+=============================================
+
+Next, configure the application using the files and info retrieved in the 
+previous steps. To do so, connect to the Tor network on the
+*Admin Workstation*, open a Terminal and run the following commands:
+
+.. code:: sh
+
+ cd ~/Persistent/securedrop
+ ./securedrop-admin setup
+ ./securedrop-admin sdconfig
+
+.. note:: The ``./securedrop-admin setup`` command may take several minutes to complete, and may
+ fail due to network issues. If it fails, it's safe to run again.
+
+When prompted, fill in the values retrieved in the previous steps.
+
+Once complete, configure the *Admin Workstation* using the command:
+
+.. code:: sh
+
+  ./securedrop-admin tailsconfig
+
+This will set up desktop shortcuts for the *Source* and *Journalist Interfaces*,
+and configure adminstrative access to the servers.
+
+Once the ``./securedrop-admin tailsconfig`` command is complete, verify that you can connect to 
+the servers using ``ssh app`` and ``ssh mon``, accepting the host verification
+prompt if necessary.
+
+Next, back up the servers by running the following command in the terminal:
+
+.. code:: sh 
+ 
+ ./securedrop-admin backup
+
+
+Step 6: Run the ``./securedrop-admin install`` command
+======================================================
+
+After the ``./securedrop-admin backup`` command completes successfully, you should  undo the
+changes made to enable temporary local SSH access, by running the following 
+command:
+
+.. code:: sh
+
+ ./securedrop-admin install
+
+Once the command completes successfully, your new *Admin Workstation* is ready
+for use.
+
+Step 7: Post-rebuild tasks
+==========================
+
+We recommend completing the following tasks after the rebuild:
+
+ - Set up a new administration account on the *Journalist Interface*, by following
+   :doc:`these instructions <create_admin_account>`
+ - Verify that submissions can be decrypted, by going through the decryption 
+   workflow with a new submission.
+ - Back up your *Admin Workstation* using the process 
+   :ref:`documented here <backup_workstations>`.
+ - Delete invalid admin accounts in the *Journalist Interface*.
+ - Delete invalid SSH public keys from the administration shell account on the
+   *Application* and *Monitor Servers*.
```

Comment:
I think we should explain how to do this part (removal of invalid SSH public keys).

Reply:
Same, but I'd put it in a separate doc describing off-boarding admins, linked to from here once it exists. I thought about adding a command to remove the authorized_keys files from the servers early on, but that might break access for legit admins.

Reply:
I agree a separate doc would be great, but perhaps we can add a bit more detail here until we have that? Revoking invalid SSH accounts feels like a crucial part of the process; IMO we should give less experienced admins a bit more guidance here (including caveats about the risks).

Reply:
This is actually a pretty straightforward candidate for Ansiblization, excuse the temporary lint mess while I see about adding a playbook to clean up authorized_keys.

Reply:
Added a securedrop-admin command and some more docs to address this - see updated test plan.